stampifier.py

```diff
@@ -63,6 +87,23 @@ def get_stampified_content(self):
             "stamp_pages": self.stampified_pages
         }
 
+    def _preprocess_contents(self):
+        '''
+        This method will use ExtractorOutputPreprocessor
+        to split the contents into different types
+        '''
+        output_preprocessor \
+            = ExtractorOutputPreprocessor(self._website.contents)
+        self.preprocessed_contents_dict \
+            = output_preprocessor.get_preprocessed_content_lists()
+        self.title_text_contents \
+            = self.preprocessed_contents_dict["titles"]
+        self.normal_text_contents \
+            = self.preprocessed_contents_dict["sentences"]
+        self.media_contents = self.preprocessed_contents_dict["media"]
+        self.embedded_contents \
+            = self.preprocessed_contents_dict["embedded_content"]
```

Comment:
@telescopic Why did this dic and not class (as in `data_model/PreprocessedContents`)? Why not pass entire PreprocessedContents object to Classifier, and Summarizer?

Reply:
Ah yeah we could actually. Raised an issue for this! Will raise a PR for this once this PR is merged.
Hope this is alright! Thanks for the suggestion!

Reply:
Resolving this conversation!